test/functional/JIT_Test/playlist.xml

```diff
@@ -664,7 +664,7 @@
 		<variations>
 			<variation>-Xint</variation>
 			<variation>-Xjit:count=1,disableAsyncCompilation</variation>
-			<variation>-Xjit:count=1,disableAsyncCompilation -XX:+CompactStrings</variation>
+			<variation>-Xjit:count=1,disableAsyncCompilation -XX:+CompactStrings -Xnoaot</variation>
```

Comment:
If we are investigating a crash, then specifying `-Xnoaot` makes life easier when it comes to collect diagnostic information.

Since AOT is enabled by default now, many methods are `AOT` compiled by default. The command line options processing when it comes to this area can be confusing. AOT compiles pick up some options specified via `-Xjit` and doesn't pickup others. 

I've run into situations where specifying JIT options the regular way to collect trace logs doesn't work when a method is AOT compiled. We instead have to specify `-Xaot:<options>` on the java command line. This leads to some churn before I realize that the trace logs are not being created because the method was AOT compiled. IIRC specifying `export TR_Options=` doesn't works either. Last I checked these issues weren't fixed either. But I could be wrong.

If we have this `-Xnoaot` option available, then exporting `TR_Options=` to quickly grab a trace log without modifying the test scripts can be done. The evaluator also has no special AOT guards, so the compiles should largely be the same whether it's enabled or not.

Reply:
Ok, thanks so this is for debugging purposes and not anything related to functionality of the implementation here that is being tested. I'll leave this review as unresolved for future readers.

Reply:
I added another variation so we test with AOT enabled as well.